tools/state-viewer/src/tx_dump.rs

```diff
@@ -0,0 +1,48 @@
+use near_chain::ChainStore;
+use near_chain::ChainStoreAccess;
+use near_primitives::account::id::AccountId;
+use near_primitives::block::Block;
+use near_primitives::transaction::SignedTransaction;
+
+/// Returns a list of transactions found in the block.
+pub fn tx_dump(
+    chain_store: &mut ChainStore,
```

Comment:
do you need mut?

Reply:
Good catch. Looks like it compiles without the `mut`.